modules/icgc-argo-workflows/prep/metrics/resources/usr/bin/prealn.py

```diff
@@ -206,25 +175,9 @@ def main():
 
     mqc_stats_updated = {k: v for k, v in mqc_stats.items() if v}
 
-    with open("%s.argo_metrics.json" % (args.sampleId), 'w') as f:
+    with open("%s.prealn_argo_metrics.json" % (args.sampleId), 'w') as f:
```

Comment:
You should use `%s.prealn.argo_metrics.json`, , change the `_` to `.`
Otherwise cause the error in tests:
```
Missing output file(s) `*.argo_metrics.json` expected by process
```